python/cudf/cudf/tests/test_array_ufunc.py

```diff
@@ -47,6 +51,21 @@ def _hide_ufunc_warnings(ufunc):
                 category=RuntimeWarning,
             )
             yield
+    elif name in {
+        "bitwise_and",
+        "bitwise_or",
+        "bitwise_xor",
+    }:
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                "ignore",
+                "Operation between non boolean Series with different "
```

Comment:
Just to confirm, this is because a cudf operations is being compared to the equivalent pandas operation, and that pandas operation triggers this behavior?

Reply:
Yup

Reply:
Gotcha and the plan will be to deprecate the similar behavior in cudf in the future right?

Reply:
> Gotcha and the plan will be to deprecate the similar behavior in cudf in the future right?

Yes, that's right.